ecosystem/sep-0038.md

```diff
@@ -552,17 +552,19 @@ must also provide either `sell_delivery_method` or `buy_delivery_method`, but no
-| `sell_asset`  | string                                                                        | The asset the client would like to sell. Ex. `USDC:G...`, `iso4217:ARS`                                                                                                                                                                                                                  |
-| `sell_amount` | string                                                                        | The amount of `sell_asset` to be exchanged for `buy_asset`. It could be different from the `sell_amount` provided in the request, depending on how fees are applied by the Anchor.                                                                                                       |
-| `buy_asset`   | string                                                                        | The asset the client would like to exchange for `sell_asset`.                                                                                                                                                                                                                            |
-| `buy_amount`  | string                                                                        | The amount of `buy_asset` to be exchanged for `sell_asset`. It could be different from the `buy_amount` provided in the request, depending on how fees are applied by the Anchor. `price * buy_amount = sell_amount` must be true up to the number of decimals required for `buy_asset`. |
-| `fee`         | object                                                                        | An object describing the fee used to calculate the conversion price. This can be used to datail the price components for the end-user.                                                                                                                                                   |
+| Name                   | Type                                                                          | Description                                                                                                                                                                                                                                                                              |
+| ---------------------- | ----------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Some modifications do not seem necessary though. It would be nice to change just the `buy_delivery_method`

Reply:
Same as above. To fix the broken table